portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextArea.path

```diff
@@ -25,6 +25,11 @@
 	<td>//*[normalize-space(text())='${key_label}']/following-sibling::textarea | //div[normalize-space()='${key_label}']/following-sibling::textarea</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ANY_LABELED_OPTIONAL</td>
+	<td>//*[@class="ddm-field-text form-control"]</td>
+	<td></td>
```

Comment:
Hey @eltonccdantas Could you use a more specific path? Something that gets a variable making it clearer what part of the page is being used.